docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: "3.9"
+
+services:
+  app:
+    build:
+      context: .
+    ports:
+      - "8000:8000"
+    env_file:
+      - .env
+    depends_on:
+      - db
+    volumes:
+      - ./:/app
+
+  db:
+    image: postgres:14-alpine
+    ports:
+      - "5433:5432"
+    env_file:
+      - .env
+
+  wait-for-db:
+    image: wait-for-it
```

Comment:
The `wait-for-it` image is used to wait for the database to be ready. Ensure that this image is available and correctly configured in your Docker environment. Alternatively, consider using a custom script if more control is needed.